main.yml

```diff
@@ -1,12 +1,20 @@
 - name: Main
-  hosts: '{{ target }}'
+  hosts: "{{ target }}"
   gather_facts: false
   vars_files:
-    - '{{ var_file }}'
+    - "{{ var_file }}"
   roles:
     - node_install
     - node_initialize
     - node_configure
     - node_launch
     - role: support_sync_snapshot
-      when: snapshot_url is defined
\ No newline at end of file
+      when: snapshot_url is defined
+
+- name: Setup Prometheus Monitoring on Chain
+  hosts: telemetry
+  vars_files:
```

Comment:
explicitly set it to gather_facts: false. Default is true.

Reply:
done.